scripts/run_qemu.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+qemu-system-x86_64 \
+    -d 'cpu_reset' \
+    -enable-kvm \
+    -s \
+    -nographic \
+    -netdev user,id=wan,hostfwd=tcp::2223-10.0.2.15:22 \
+    -device virtio-net-pci,netdev=wan,addr=0x06,id=nic1 \
+    -netdev user,id=lan,hostfwd=tcp::6080-192.168.1.1:80,hostfwd=tcp::2222-192.168.1.1:22,net=192.168.1.100/24 \
+    -device virtio-net-pci,netdev=lan,addr=0x05,id=nic2 \
```

Comment:
In my qemu script I also have the following lines:
```
        -chardev stdio,id=char0,signal=off \
        -serial chardev:char0 \
```
This connects the serial console of the VM to the terminal running the script. `signal=off` might be controversial - it makes `Ctrl-C` etc. work as expected in the serial console instead of terminating qemu, but in consequence you need a second terminal if you actually want to kill qemu.

Reply:
The proper way to exit qemu with `signal=off` is to press `Ctrl-a`, `Ctrl-c` and type `quit`.

Reply:
Good to know, I wasn't aware qemu had an escape key like that.

Reply:
I added a note to `run_qemu.sh`, so hopefully other users will stumble across that hint aswell.